README.md

```diff
@@ -85,34 +85,31 @@ rafterydiag(c::AbstractChains; q=0.025, r=0.005, s=0.95, eps=0.001)
 ### Plotting
 ```julia
 # construct a plot
-plot(c::AbstractChains, type::Symbol)
+plot(c::AbstractChains; ptypes = [TracePlot, MixedDensityPlot])
```

Comment:
To help downstream packages, it might be helpful to add an alias function to keep supporting the old syntax: `plot(c::AbstractChains, :density)`?

Reply:
I was wrong. In fact, `Plots.jl` depends on `RecipesBase.jl` which exports `plot` and `plot!`. To support the old syntax we have to provide alias functions and export them. I'll push the code asap.